junipervpn/vpn.py

```diff
@@ -310,46 +311,45 @@ def stop(self):
 
 def main():
     parser = argparse.ArgumentParser(conflict_handler='resolve')
-    parser.add_argument('-h', '--host', type=str,
+    parser.add_argument('-h', '--host',
                         help='VPN host name')
-    parser.add_argument('-r', '--realm', type=str,
+    parser.add_argument('-r', '--realm',
                         help='VPN realm')
-    parser.add_argument('-u', '--username', type=str,
+    parser.add_argument('-u', '--username',
                         help='User name')
-    parser.add_argument('-p', '--pass_prefix', type=str,
+    parser.add_argument('-p', '--pass_prefix',
                         help="Secondary password prefix")
-    parser.add_argument('-o', '--oath', type=str,
+    parser.add_argument('-o', '--oath',
                         help='OATH key for two factor authentication (hex)')
-    parser.add_argument('-c', '--config', type=str,
+    parser.add_argument('-c', '--config',
                         help='Config file')
-    parser.add_argument('-s', '--stdin', type=str,
+    parser.add_argument('-s', '--stdin',
                         help="String to pass to action's stdin")
-    parser.add_argument('-d', '--device-id', type=str,
+    parser.add_argument('-d', '--device-id',
                         help="Hex device ID")
     parser.add_argument('-f', '--enable-funk', action='store_true',
+                        default=None,
```

Comment:
OK - this has to be `None`, rather than `False`, because it's handled in a function which doesn't know if it's come from `argparse` or a config file (where it could be a string like `"no"`).